README.md

```diff
@@ -23,3 +25,5 @@ go get github.com/equinix/equinix-sdk-go
 ## Usage
 
 You can see usage of the generated code in the [`examples` directory](https://github.com/equinix/equinix-sdk-go/tree/main/examples).
+
+Full package documentation is available at <https://pkg.go.dev/github.com/equinix/equinix-sdk-go/services/metalv1>.
```

Comment:
Rather than linking to a specific service, this should be a link to the top level docs for the module